sandboxes/cf.txt

```diff
@@ -1,5 +1,6 @@
 # version 11
 
+tenacity==8.3.0
```

Comment:
Instead of pinning a specific version, I would just exclude the version that is known to fail.

```suggestion
tenacity!=8.4.0
```